coordinator/src/services/translation/request/organization.ts

```diff
@@ -117,7 +109,7 @@ function convertCommonOrganizationDetails(
     `${costCentre.resourceType}.identifier`
   )
   result.id = new hl7V3.SdsOrganizationIdentifier(organizationSdsId)
-  result.code = new hl7V3.OrganizationTypeCode(organisationTypeCode)
+  result.code = new hl7V3.OrganizationTypeCode(OrganisationTypeCode.NOT_SPECIFIED)
```

Comment:
(Optional) Consider adding a comment referencing the ticket - will likely help people looking at the code in the future

Reply:
Would you want the comment referencing ticket/decision log on the same line? Line above?

Reply:
Up to you, whichever looks best 😄

Reply:
Added a comment pointing to the ticket!